config/initializers/devise.rb

```diff
@@ -250,6 +250,8 @@
   # ==> OmniAuth
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
+  config.omniauth :facebook, '1957979147801504', '6e06570424d211750ca738deb028a9ee',
```

Comment:
You shouldn't publish your keys in github, this is private data. Use `figaro` gem instead (check https://github.com/laserlemon/figaro)

You should create a file using the gem, for example:
```
# config/application.yml

FACEBOOK_APP_ID: "YOUR_APP_ID"
FACEBOOK_APP_KEY: "YOUR_FACEBOOK_APP_KEY"
```

And use it as: `ENV['FACEBOOK_APP_ID']`